components/logged_in_navigation/components/notifications/templates/notification.jade

```diff
@@ -0,0 +1,37 @@
+- klasses = [];
+
+- group.isNew() ? klasses.push('Notification--unread') : '';
+- group.is_comment() ? klasses.push('Notification--comment') : '';
```

Comment:
Mm it's got a bit of a different format, but it should look more or less the same:
![screenshot 2017-05-26 17 30 58](https://cloud.githubusercontent.com/assets/821469/26513375/24cef31e-4239-11e7-9446-c3ffa3a865fc.png)